packages/mdc-chips/README.md

```diff
@@ -409,7 +409,7 @@ Method Signature | Description
 --- | ---
 `isSelected() => boolean` | Returns true if the chip is selected
 `setSelected(selected: boolean) => void` | Sets the chip's selected state
-`setSelectedFromChipSet(selected: boolean) => void` | Sets the chip's selected state (called from the chip set)
+`setSelectedFromChipSet(selected: boolean, shouldNotify: boolean) => void` | Sets the chip's selected state (called from the chip set) to the `selected` param. Will emit a selection event if called with `shouldNotify` set to `true`
```

Comment:
Can `shouldNotify` be same as param name in `notifySelection` (ie., `chipSetShouldIgnore`)? Boolean flag is reversed in both these methods and it's confusing.

Or Can we rename all these booleans to `isUserSelected` or `selectOnAction`? This boolean will be set to true only when it is user selected. WDYT?

Reply:
I'll go with the first option. Thanks!

Reply:
The `shouldNotify` is for clients. When `setSelectedFromChipSet` is called, the event is ignored by the chip set. I'll update the param to make it clear it's for client notification.

Reply:
We still use different terminology. It would be great if we can align the terminology for this boolean flag across component. (`chipSetShouldIgnore`, `shouldNotify`, `chipSetShouldIgnore` & `shouldNotifyClients`).

Reply:
Cool cool. I updated this so only `shouldNotifyClients` is used.